resources/aws_ec2.go

```diff
@@ -133,6 +141,12 @@ type AwsEc2Res struct {
 
 	WatchEndpoint   string `yaml:"watchendpoint"`   // the public url of the sns endpoint, eg: http://server:12345/
 	WatchListenAddr string `yaml:"watchlistenaddr"` // the local address or port that the sns listens on, eg: 10.0.0.0:23456 or 23456
+	// ErrorJSONStrict is a bool that controls weather json.Decode(post)
+	// errors are sent to the engine or not. If it is set to true and an
+	// error is produced, Watch() will return the error and the engine will
+	// shut down.
+	ErrorJSONStrict bool `yaml:"errorjsonstrict"`
```

Comment:
Let's call this something without the word JSON. (The implementation should be allowed to change without the interface needing to.) How about ErrorOnInvalidMessages or similar (replace Messages with whatever they're called)